Example/Photos/With Stores/Banner Feature/View Level/BannerUpdateViewStore.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  BannerUpdateViewStore.swift
+//  ViewStore
+//
+//  Created by Kenneth Ackerson on 4/3/23.
+//
+
+import Foundation
+import Combine
+import SwiftUI
+import CasePaths
+
+typealias BannerUpdateViewStoreType = Store<BannerUpdateViewStore.State, BannerUpdateViewStore.Action>
+
+/// A `Store` that drives a view that can update a `Banner` through any `BannerDataStoreType`, and exposes view-specific state such as a working copy of the banner, the possible networking error, etc.
+final class BannerUpdateViewStore: Store {
+    
+    // MARK: - Store
+    
+    struct State {
+        let bannerViewState: BannerDataStore.State
+        
+        let workingCopy: Banner
+        
+        var dismissable: Bool {
+            switch bannerViewState.networkState {
+            case .notStarted, .finished:
+                return true
+            case .inProgress:
+                return false
+            }
+        }
+        
+        var success: Bool {
+            switch bannerViewState.networkState {
+            case .notStarted, .inProgress:
+                return false
+            case .finished(let result):
+                return (try? result.get()) != nil
+            }
+        }
+        
+        var error: NSError? {
+            switch bannerViewState.networkState {
+            case .notStarted, .inProgress:
+                return nil
+            case .finished(let result):
+                do {
+                    _ = try result.get()
+                    return nil
+                }
+                catch {
+                    return error as NSError
+                }
+            }
+        }
```

Comment:
Regardless of the error refactor mentioned above, I think this could be reduced to one line.
```suggestion
        var error: MockBannerNetworkStateController.Error? {
            return bannerViewState.networkState.error
        }
```